domain_setup.py

```diff
@@ -0,0 +1,14 @@
+from distutils.core import setup, Extension
```

Comment:
(personal opinion) It would be better to have one `setup.py` so that installation is controlled more one place.